langchain4j-core/src/main/java/dev/langchain4j/rag/DefaultRetrievalAugmentor.java

```diff
@@ -247,8 +264,8 @@ private static void log(Map<Query, Collection<List<Content>>> queryToContents, L
                         .collect(joining("\n")));
     }
 
-    private static void log(UserMessage augmentedUserMessage) {
-        log.trace("Augmented user message: " + escapeNewlines(augmentedUserMessage.singleText()));
+    private static void log(ChatMessage augmentedChatMessage) {
+        log.trace("Augmented chat message: " + escapeNewlines(augmentedChatMessage.text()));
```

Comment:
Too Minor : Placeholder instead of concatenation
` log.trace("Augmented chat message: {}", escapeNewlines(augmentedChatMessage.text()));
`